tsconfig.shared.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "target": "es2017",
```

Comment:
Target and lib probably should not be in the shared, although it does not matter since they will be overwritten in the frontend tsconfigs.